addon/mixins/application-route-mixin.js

```diff
@@ -15,20 +17,22 @@ const { inject, on } = Ember;
 
   ```js
   // app/instance-initializers/session-events.js
-  Ember.Application.initializer({
-    name:       'session-events',
-    after:      'ember-simple-auth',
-    initialize: function(container, application) {
-      var applicationRoute = container.lookup('route:application');
-      var session          = container.lookup('service:session');
+  export function initialize(instance) {
+      var applicationRoute = instance.container.lookup('route:application');
+      var session          = instance.container.lookup('service:session');
       session.on('authenticationSucceeded', function() {
         applicationRoute.transitionTo('index');
       });
       session.on('invalidationSucceeded', function() {
         window.location.reload();
       });
-    }
   });
+
+  export default {
+    name:       'session-events',
+    after:      'ember-simple-auth',
+    initialize: initialize
+  };
```

Comment:
True. However, I think a more "complete" example would go far in helping people new to Ember/ES6 figure out how things work in relation to the rest of the ecosystem (I'm new, and this would have helped me understand it better). This way is the example of how initialisers are used I see most often, so I think it would help to keep things consistent.